src/test/resources/example-company-response.json

```diff
@@ -9,6 +9,7 @@
   "logo" : "https://d2ojpxxtu63wzl.cloudfront.net/static/1bacd7306731a30d2a9f024eeb1dcff1_94d77dcdedbfe40707ac4a75ca4f4d2978bffc20b2e33a3288ea9e4d47f5af6c",
   "website" : "https://www.fullcontact.com",
   "languageLocale" : "en",
+  "industries":[],
```

Comment:
it works OK even if there's no key at all, right? Otherwise this LGTM

Reply:
Yes since the pr builder passed when I first merged and I just tested again locally.